asteroid/metrics.py

```diff
@@ -204,10 +222,17 @@ def wer_from_hsdi(hits=0, substitutions=0, deletions=0, insertions=0):
 
     @staticmethod
     def hsdi(truth, hypothesis):
+        import jiwer
         from jiwer import compute_measures
 
+        transformation = jiwer.Compose([jiwer.ToLowerCase(), jiwer.RemovePunctuation()])
```

Comment:
Should this be defined in the constructor (init) instead?

Reply:
It should :)

Reply:
Well, if we do that we have to make `self.transformation` an argument of `hsdi()` or `hsdi()` can stay a static method

Reply:
Let's keep hsdi a static function and pass the transformation.